text/0025-core-compute-resources-as-plugins.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: Core Compute Resources as Plugins
+- Start Date: 2016-10-21
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Core Compute Resources should move to plugins
+
+# Motivation
+[motivation]: #motivation
+
+Core has been set up to allow compute resources as plugins for some while, and
+we already have several examples of this in use (Xen, Azure, DigitalOcean,
+etc). This has brought benefits:
+
+* able to release independatly of Core
+* make use of the modular Fog gems that are now available
+* separate logging of issues
+* dedicated maintainers who know the compute resource well
+
+In addition, it allows the core project to mark a particular resource as
+unmaintained when necessary (as happened with Xen). 
```

Comment:
I think I agree with you - but since I find the benefits of moving a plugin a strong argument in-and-of-itself, I would move it to a plugin anyway (and this RFC is currently written such that not all CRs will move at once). Your comment puts you in disagreement with @mmoll though, who suggests all CRs should move in one release cycle (https://github.com/theforeman/rfcs/pull/28/files#r85011715). Discuss! :)